tests/tests_pytorch/test_cli.py

```diff
@@ -65,6 +68,22 @@ def mock_subclasses(baseclass, *subclasses):
         yield None
 
 
+@pytest.fixture
```

Comment:
The CLI tests were relying on a lot of silent behaviour from our loggers. This is why I had to do this many changes to the tests.